src/apphosting/githubConnections.ts

```diff
@@ -180,6 +195,107 @@ async function createFullyInstalledConnection(
   return conn;
 }
 
+async function manageInstallation(connection: devConnect.Connection): Promise<void> {
```

Comment:
nit* it looks like "manage" is an intentional choice of word here but I was confused because I expected it to be "update" or "configure". Curious why you ended up deciding on "manage".

Reply:
I thought "manage" makes sense  because it takes them to the installation page from which users can either, update the installation's repository access, suspend the installation, or uninstall the GitHub app. I think "configure" could work here as well.

<img width="884" alt="Screenshot 2024-07-11 at 9 47 43 AM" src="https://github.com/firebase/firebase-tools/assets/14209679/520eb70f-d759-48ca-b139-8c01ed83fbe0">